pkg/client/optimizely_user_context_test.go

```diff
@@ -17,7 +17,7 @@
 package client
 
 import (
-	"io/ioutil"
+	"os"
```

Comment:
why this is changed?

Reply:
Suggested by linter as "`io/ioutil` is now depreceated in go 1.16 onwards.